bedrock/base/templates/includes/protocol/navigation/nav-cta.html

```diff
@@ -19,7 +19,7 @@
           class_name='mzp-t-product mzp-t-secondary mzp-t-md',
           optional_attributes= {
             'data-cta-text' : 'Get Mozilla VPN',
-            'data-cta-type' : 'button',
+            'data-cta-type' : 'vpn',
```

Comment:
This (and any other use of `vpn_product_referral_link`) link to a page on WMO and so should not have the `fxa-` prefix other cta-types have.